CHANGELOG/CHANGELOG-1.2.md

```diff
@@ -0,0 +1,36 @@
+# v1.2.0 - Changelog since v1.1.0
+
+## Features
+
+- Improved Windows Support
+  - Switch to CSI Proxy v1beta2 API (previously v1beta1). ([#669](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/669), [@jingxu97](https://github.com/jingxu97))
+  - Add Disk online/offline logic in nodeStageVolume/nodeUnstageVolume calls for Windows. This requires requires the CSI Proxy `disk.v1beta2` group, which is only available in CSI proxy v0.2.2+. ([#661](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/661), [@jingxu97](https://github.com/jingxu97))
+- Update kustomizer to 3.8.6. ([#661](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/661), [@jingxu97](https://github.com/jingxu97))
```

Comment:
this is not specific for Windows. Since Kustomizer is used to generate different deployment overlays for testing, maybe move it in test section?

Reply:
Note the indentation. This is not under Windows bullet point. But I'm ok moving it to testing section.